htdocs/gulpfile.js

```diff
@@ -1,5 +1,47 @@
+'use strict';
```

Comment:
This file is important for asset management. It basically runs the tasks for bundling and uglifying css and js and provides sourcemaps for debugging.